src/Arkanis.Overlay.Host.Desktop/Workers/GlobalLaunchShortcutTracker.cs

```diff
@@ -14,7 +14,7 @@ namespace Arkanis.Overlay.Host.Desktop.Workers;
 ///     Captures global hotkeys.
 ///     Based on: https://learn.microsoft.com/en-us/archive/blogs/toub/low-level-keyboard-hook-in-c
 /// </summary>
-public sealed class GlobalHotkey : IHostedService, IDisposable
+public sealed class GlobalLaunchShortcutTracker : IHostedService, IDisposable
```

Comment:
Maybe `GlobalHotkeyListener` or `GlobalShortcutListener` or `GlobalKeyboardShortcutListener` might be better, since in the future we will want to add multiple hotkeys / keyboard shortcuts and support listening to them to directly invoke specific functionality, e.g. open overlay with a certain module / page active directly.